### ISSUE TITLE
`bicycle_road=yes` on `highway=cycleway` not rendered

### ISSUE THREAD

**Issue by tordans** [cyclosm/cyclosm-cartocss-style#629]
I am wondering, why the bottom segment https://www.cyclosm.org/#map=18/52.48459/13.44465/cyclosm of Weigandufer is not blue. It should at least be a blue line because of the `highway=cycleway`, right? But even more, IMO the whole road width should be blue because of the `bicycle_road=yes`. Right now, the safest part of that road is not shows as bike infrastructure :).

- top part https://www.openstreetmap.org/way/744158640#map=19/52.48483/13.44414
- bottom part https://www.openstreetmap.org/way/859427215#map=18/52.48412/13.44547 (the issue here)
- mapillary https://www.mapillary.com/app/?pKey=1287704951654498&focus=photo

<img width="694" alt="image" src="https://user-images.githubusercontent.com/111561/185693266-e6da7c10-51c2-4864-b01a-ee11e20798a8.png">

--

Follow up to https://github.com/cyclosm/cyclosm-cartocss-style/issues/150

**Comment by Florimondable** [comment]
It's due to the tag vehicle=no, a bicycle is a vehicle, so the cycleway is rendered in grey as not allowed for bicycle.

About bicycle_road it's weird to have this tag on a cycleway, though after seeing mapillary I can understand why : there is the Fahrradstraße board.

**Comment by tordans** [comment]
Thanks for helping with debugging this. We discussed it locally and found, that we broke the tagging at the beginning of the year. [History](https://osmlab.github.io/osm-deep-history/#/way/859427215). The segment is now blue again. 

I would still argue, that anything with bicycle_road=yes should be shown more like a highway=service or highway=residential that a "thin", regular highway=cycleway. In my mind, the "bicycle_road" tagging could just as easily be "highway=bicycle_road" which probably did not happen historically because we don't change the highway-schema anymore. But for a full bicycle_road (aka no other traffic than bikes), the highway=cycleway + bicycle_road=yes feels like the best tagging to describe the function of the street.

Feel free to close if you don't want to change this.
